package.json

```diff
@@ -85,57 +82,61 @@
     "lodash.escaperegexp": "^4.1.2",
     "lodash.kebabcase": "^4.1.1",
     "lodash.startcase": "^4.4.0",
-    "postcss": "^6.0.21",
     "prop-types": "^15.5.8",
     "react-intl": "^2.4.0",
-    "terra-base": "^5.0.0",
+    "react-router-dom": "^5.0.0",
```

Comment:
My builds weren't successful unless I added this myself. We should keep this as a peer

Reply:
Though it does look like your framework PR doesn't have it.. 🤔